packages/expo-clipboard/README.md

```diff
@@ -7,7 +7,7 @@
   </a>
 </p>
 
-`expo-clipboard` provides an interface for getting and setting Clipboard content on Android, iOS, and Web.
```

Comment:
we use oxford comma, so the comma here is correct. https://www.notion.so/expo/Writing-Guide-35a1eb87298a49c2ad80f8a4f21b7c52#69050271bd0f499e81ea432ab1b42c77